src/System.Data.Odbc/src/System/Data/Odbc/OdbcException.cs

```diff
@@ -45,6 +53,8 @@ public OdbcErrorCollection Errors
         public override void GetObjectData(SerializationInfo si, StreamingContext context)
         {
             base.GetObjectData(si, context);
+            si.AddValue("odbcRetcode", (ODBC32.RETCODE)100, typeof(ODBC32.RETCODE)); // NO DATA
```

Comment:
Why 100? Desktop will default to 0

Reply:
0 = SUCCESS
100 = NO_DATA

I thought of 100 as no_data to provide to the user but maybe 100 is known as there is no_data at the odbc endpoint... I'm not really happy with both...

Reply:
Should have a comment ideally then

Reply:
I will change it to 0 and add a comment.